packages/eas-cli/src/commands/update/index.ts

```diff
@@ -391,45 +400,53 @@ export default class UpdatePublish extends EasCommand {
 
       // build bundle and upload assets for a new publish
       if (!skipBundler) {
-        const bundleSpinner = ora().start('Building bundle...');
+        const bundleSpinner = ora().start('Exporting...');
         try {
           await buildBundlesAsync({ projectDir, inputDir });
-          bundleSpinner.succeed('Built bundle!');
+          bundleSpinner.succeed('Exported platforms');
```

Comment:
This should say something about exporting updates, not platforms. Also, if you decide to include the word "platforms" in the message, make sure to render it as "platform" singular when there's just one target output platform.